samples/js/initializers.md

```diff
@@ -1,31 +1,36 @@
-application.coffee:
+in `app/client/packs/application.js` (or another file that is always loaded)
 
-```coffee
-...
-$ ->
-  $body = $('body')
-  new window.Initializers($body)
+```js
+$(document).ready(() => new Initializer($('body')).initialize());
 
-  $body.on 'cocoon:after-insert' ($newField) ->
-    new window.Initializers($newField)
 ```
 
-initializers.coffee:
 
-```coffee
-class window.Initializers
-  constructor: ($scope) ->
-    @$scope = $scope
-    @select2
-    @bestInPlace
-    @datePicker
-    @tooltip
+in `initializer.js`:
 
-  select2: ->
-    @$scope.find('.select2').select2
-      allowClear: true
-      placeholder: 'Select Option'
+```js
+export default class Initializer {
+  constructor($scope) {
+    this.$scope = $scope;
+  }
 
-  tooltip: ->
-    @$scope.find("[data-toggle='tooltip']").tooltip()
+  initialize() {
+    this.initializeTooltips();
+    this.initializeDatePickers();
+  }
+
+  initializeTooltips() {
+    const $tooltips = this.$scope.find('[data-content]');
+    $tooltips.each((index, element) => {
```

Comment:
I just ran this example through the eslint config for MGP, and it reported

> 'index' is defined but never used. Allowed unused args must match /^_/. (no-unused-vars)

Not sure if that's just the MGP config or something we like having across projects (it at least matches up with Rubocop). If it's something you think is worth changing, the change should be replicated on the `$datePickers.each` line below.